packages/core/src/fallback/handler.ts

```diff
@@ -36,10 +37,12 @@ export async function handleFallback(
   ) {
     return null;
   }
-
+  const shouldSetPreviewBypassMode =
```

Comment:
nit: should it be !(error instanceof TerminalError) that way we normally attempt the fallback to 2.5 Pro